all-actors/src/main/java/org/sunbird/NotificationValidator.java

```diff
@@ -0,0 +1,81 @@
+package org.sunbird;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.sunbird.pojo.NotificationMode;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.message.IResponseMessage;
+import org.sunbird.message.ResponseCode;
+import org.sunbird.pojo.NotificationRequest;
+import org.sunbird.request.Request;
+
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Validates send notification api request
+ */
+public class NotificationValidator {
+
+
+    public static void validateSendNotificationRequest(Request request) throws BaseException {
```

Comment:
Does this need Request or array of notification objects? Correct accordingly.

Reply:
actually I am mapping this request to pojo, in this method